scripts/deploy-app.sh

```diff
@@ -46,57 +53,27 @@ generateImageArgs(){
 }
 VRO_IMAGE_ARGS=$(generateImageArgs "${ENV}" "${IMAGE_TAG}")
 
-if [ "${ENV}" == "qa" ] || [ "${ENV}" == "dev" ]
-then
+COMMON_HELM_ARGS="--set-string environment=${ENV} \
+--set-string info.version=${IMAGE_TAG} \
+--set-string info.git_hash=${GIT_SHA} \
+--set-string info.deploy_env=${ENV} \
+--set-string info.github_token=${GITHUB_ACCESS_TOKEN} \
+\
+--set-string images.redis.imageName=redis \
+--set-string images.redis.tag=latest \
+\
+--set-string images.mq.imageName=rabbitmq \
+--set-string images.mq.tag=3 \
+"
+
 : "${TEAMNAME:=va-abd-rrd}"
 : "${HELM_APP_NAME:=abd-vro}"
-helm del $HELM_APP_NAME -n ${TEAMNAME}-${ENV}
+# K8s namespace
+NAMESPACE="${TEAMNAME}-${ENV}"
+
+helm del $HELM_APP_NAME -n ${NAMESPACE}
 helm upgrade --install $HELM_APP_NAME helmchart \
-              --set-string environment=${ENV} \
-              --set-string info.version=${IMAGE_TAG} \
-              --set-string info.git_hash=${GIT_SHA} \
-              --set-string info.deploy_env=${ENV} \
-              --set-string info.github_token=${GITHUB_ACCESS_TOKEN} \
-              --set-string images.redis.tag=latest \
-              --set-string images.redis.imageName=redis \
-              --set-string images.mq.tag="3" \
-              --set-string images.mq.imageName=vro-rabbitmq \
-              ${VRO_IMAGE_ARGS} \
+              ${COMMON_HELM_ARGS} ${VRO_IMAGE_ARGS} \
               --debug \
-              -n ${TEAMNAME}-${ENV} #--dry-run
+              -n ${NAMESPACE} #--dry-run
               #-f helmchart/"${ENV}".yaml
-elif [ "${ENV}" == "sandbox" ] || [ "${ENV}" == "prod" ] || [ "${ENV}" == "prod-test" ]
-then
-: "${TEAMNAME:=va-abd-rrd}"
-: "${HELM_APP_NAME:=abd-vro}"
-helm del $HELM_APP_NAME -n ${TEAMNAME}-"${ENV}"
-helm upgrade --install $HELM_APP_NAME helmchart \
-              --set-string images.repo=abd-vro-internal \
-              --set-string environment="${ENV}"\
-              --set-string info.version="${IMAGE_TAG}"\
-              --set-string info.git_hash="${GIT_SHA}" \
-              --set-string info.deploy_env="${ENV}" \
-              --set-string info.github_token="${GITHUB_ACCESS_TOKEN}" \
-              --set-string images.redis.tag="latest"\
-              --set-string images.redis.imageName=redis \
-              --set-string images.mq.tag="3"\
-              --set-string images.mq.imageName=rabbitmq \
-              ${VRO_IMAGE_ARGS} \
-              --debug \
-              -n ${TEAMNAME}-"${ENV}"
```

Comment:
These two blocks have been consolidated. The only difference is captured by `$USE_SECREL_IMAGES`.